medusa/init/filesystem.py

```diff
@@ -8,6 +8,8 @@
 import sys
 import tarfile
 
+# unused import but needs to be monkeypatched here
+import tempfile
```

Comment:
# pylint: disable=unused-import

Reply:
I don't think Travis/flake8 read that

Reply:
Well allot of IDE's do. So imo it doesn't do harm. And it prevents someone from removing it in the future, who think.. "hey an unused import". Putting the comment inline with pylint: makes it less user error prone.

Reply:
fixed for both